### ISSUE TITLE
Textmate syntax is not always working

### ISSUE THREAD

**Issue by coderbuilds** [ayecue/monaco-textmate-provider#3]
It seems that the tokenization result is not always the same, looks like there is some kind of async issue?

Attaching also a video with the issue:
https://www.awesomescreenshot.com/video/25720093?key=d723bc39ec72d2d02e8c3cce6cdea30d

The only adjustment I made is in the index.tsx from the example folder:
```javascript
const model = resolvedMonaco.editor.createModel(
      'function tester() { \n console.log("hello world"); \n}',
      'javascript'
);
``` 
Any idea what could be causing the issue?

**Comment by ayecue** [comment]
Hey thanks for reporting. Indeed it seems that the approach of injecting CSS can potentially lead to issues when `.monaco-colors` isn't available which as you said happens async and not always with the same timing. I've prepared a small fix https://github.com/ayecue/monaco-textmate-provider/pull/4 from what I've tested it seems to work fine.

I'll test a bit around myself and probably going to merge it tomorrow depending on if there are any bugs remaining.

**Comment by coderbuilds** [comment]
Thank you for checking on the issue so quickly! I did some testing too and it works fine on your PR. 
Looking forward to update to the next version tomorrow.

**Comment by ayecue** [comment]
I did some minor changes since I wasn't a huge fan of using an observer. It's now listening for the editor creation which works fine since essentially the monaco stylesheet gets created when the editor gets created.

Anyway I've published version `1.3.0` now. I'll close the issue for now if any issues occur for you feel free to reopen the ticket. Thanks 🙂

**Comment by coderbuilds** [comment]
Thanks for having a look. Good to see you cleaned up the observer 👍
